core/payment/examples/cancel_invoice.rs

```diff
@@ -0,0 +1,176 @@
+use bigdecimal::BigDecimal;
+use chrono::Utc;
+use std::time::Duration;
+use ya_client::payment::{PaymentProviderApi, PaymentRequestorApi};
+use ya_client::web::WebClient;
+use ya_client_model::payment::{
+    Acceptance, DocumentStatus, EventType, NewAllocation, NewDebitNote, NewInvoice,
+};
+use ya_core_model::payment::local as pay;
+use ya_service_bus::typed as bus;
+
+async fn assert_requested_amount(
+    payer_addr: &str,
+    payee_addr: &str,
+    payment_platform: &str,
+    amount: &BigDecimal,
+) -> anyhow::Result<()> {
+    let payer_status = bus::service(pay::BUS_ID)
+        .call(pay::GetStatus {
+            platform: payment_platform.to_string(),
+            address: payer_addr.to_string(),
+        })
+        .await??;
+    assert_eq!(&payer_status.outgoing.requested, amount);
+
+    let payee_status = bus::service(pay::BUS_ID)
+        .call(pay::GetStatus {
+            platform: payment_platform.to_string(),
+            address: payee_addr.to_string(),
+        })
+        .await??;
+    assert_eq!(&payee_status.incoming.requested, amount);
```

Comment:
Does it crash the daemon? Do we want this?

Reply:
No, it doesn't. That's just test code. The daemon is running in another process.